.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should not be part of the pull request. Please remove and add this to your .gitignore file